### PR TITLE
[fix]: [CCM-15018]: Introducing some more FAQ's for recommendations

### PR DESCRIPTION
**Pull request by shashwat-harness**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by trunapushpa — APPROVED** [review]

File: kb/cloud-cost-management/cloud-cost-management-faqs.md [hunk 1]
```diff
@@ -251,10 +251,61 @@ Also, do not update `limit` of CPU and Memory manually based on recommended `req
...
 
+#### We are not able to see recommendations section in some perspectives ?
 
+Currently, recommnedations section is only visible in perspective made through clusters
 
+#### When I create a JIRA ticket from a recommendation, will that recommendations automatically come in the applied recommendations tab once the JIRA status of the ticket is set to done ?
+
+Yes, as soon as you change the JIRA status of a ticket created for a partiucular recommnedation and change it's status to done, that particular recommendation will automatically start appearing in applied recommendations tab
+
+#### Do we provide the facility to add custom policies to get Asset Optimization Recommendation ?
```
Comment (iamanmolm) [inline kb/cloud-cost-management/cloud-cost-management-faqs.md:266]: We can change get to generate. And also change Asset Optimization Recommendation to Asset Governance Recommendation.

File: kb/cloud-cost-management/cloud-cost-management-faqs.md [hunk 2]
```diff
@@ -251,10 +251,61 @@ Also, do not update `limit` of CPU and Memory manually based on recommended `req
...
+
+Yes we do provide the facility to add custom policies to get Asset Optimization Recommendation. You can connect with our team and let us know the policies, we will hook those policies into our backend to generate recommnedations.
```
Comment (iamanmolm) [inline kb/cloud-cost-management/cloud-cost-management-faqs.md:268]: Even in the answer change the Asset Optimization Recommendation to Asset Governance Recommendation.

Comment (iamanmolm) [inline kb/cloud-cost-management/cloud-cost-management-faqs.md:268]: Even in answer recommendations spell is incorrect.

File: kb/cloud-cost-management/cloud-cost-management-faqs.md [hunk 3]
```diff
@@ -251,10 +251,61 @@ Also, do not update `limit` of CPU and Memory manually based on recommended `req
...
+
+#### Do we support any method to retrieve Kafka recommendations ?
+
+As of now we don't support any method to retrieve Kafka recommendations 
+
+#### Can we recommend cost savings by changing instance architecture e.g moving from amd64 to arm64 based instances? 
+
+If this is for nodepool reccomnedations, you need to add preferred instance families to the x86 families that you would actually use.
```
Comment (iamanmolm) [inline kb/cloud-cost-management/cloud-cost-management-faqs.md:276]: Here recommendation spell is incorrect.

Comment (iamanmolm) [inline kb/cloud-cost-management/cloud-cost-management-faqs.md:276]: Confirm this answer with @abhinavsingh-ccm

File: kb/cloud-cost-management/cloud-cost-management-faqs.md [hunk 4]
```diff
@@ -251,10 +251,61 @@ Also, do not update `limit` of CPU and Memory manually based on recommended `req
...
 
+#### Do we support showing GCP reccomendations ?
 
+As of now we don't support showing GCP recommnedations
 
+#### We are not able to see recommendations section in some perspectives ?
 
+Currently, recommnedations section is only visible in perspective made through clusters
 
+#### When I create a JIRA ticket from a recommendation, will that recommendations automatically come in the applied recommendations tab once the JIRA status of the ticket is set to done ?
```
Comment (iamanmolm) [inline kb/cloud-cost-management/cloud-cost-management-faqs.md:262]: Change "will that recommendations" to "will that recommendation".

File: kb/cloud-cost-management/cloud-cost-management-faqs.md [hunk 5]
```diff
@@ -251,10 +251,61 @@ Also, do not update `limit` of CPU and Memory manually based on recommended `req
...
+
+Yes, as soon as you change the JIRA status of a ticket created for a partiucular recommnedation and change it's status to done, that particular recommendation will automatically start appearing in applied recommendations tab
```
Comment (iamanmolm) [inline kb/cloud-cost-management/cloud-cost-management-faqs.md:264]: particular spell is incorrect.

Comment (iamanmolm) [reply kb/cloud-cost-management/cloud-cost-management-faqs.md:264]: recommendation spell is incorrect.

Comment (iamanmolm) [reply kb/cloud-cost-management/cloud-cost-management-faqs.md:264]: Confirm this with @trunapushpa, I think it would be done via batch job not instantly.

**Review by iamanmolm — CHANGES_REQUESTED** [review]

File: kb/cloud-cost-management/cloud-cost-management-faqs.md [hunk 6]
```diff
@@ -251,10 +251,61 @@ Also, do not update `limit` of CPU and Memory manually based on recommended `req
 
 Right now, the policy only identifies potential RDS instances for resize. Earlier we had a resize action associated with the policy but since it is not currently supported by custodian, we have removed it.
 
+#### Do we support showing GCP reccomendations ?
```
[the remark this answers is not in the document]
Comment (shashwat-harness) [reply kb/cloud-cost-management/cloud-cost-management-faqs.md:254]: Ohh sorry, I created these FAQs in a text editor and as a result, the word 'recommendation' may have been auto-corrected to this throughout the FAQs

File: kb/cloud-cost-management/cloud-cost-management-faqs.md [hunk 7]
```diff
@@ -251,10 +251,61 @@ Also, do not update `limit` of CPU and Memory manually based on recommended `req
...
+
+#### Can we recommend cost savings by changing instance architecture e.g moving from amd64 to arm64 based instances? 
+
+If this is for nodepool reccomnedations, you need to add preferred instance families to the x86 families that you would actually use.
+
+#### Do we support showing RDS recommnedations ?
+
+Yes, we show EC2 and ECS recommendations via native recommendations and RDS recommendation from asset governance
+
+#### Can I generate recommendation related to some resources based on expected demands in the future to rightsize up/down effectively ?
```
[the remark this answers is not in the document]
Comment (shashwat-harness) [reply kb/cloud-cost-management/cloud-cost-management-faqs.md:282]: This was a question asked by one of the customer. It makes sense considering customer wants to rightsize his resources up/down based on the future demands.

[the remark this answers is not in the document]
Comment (iamanmolm) [reply kb/cloud-cost-management/cloud-cost-management-faqs.md:282]: It is not very clear what cloud provider or resource type we are talking about?

File: kb/cloud-cost-management/cloud-cost-management-faqs.md [hunk 8]
```diff
@@ -251,10 +251,61 @@ Also, do not update `limit` of CPU and Memory manually based on recommended `req
...
+
+Yes, you can set instance family preferences for ec2 recommendations.
+
+#### Can we apply "preferred instance families" across all recommendations ?
+
+No, you would have to manually apply preferred instance families for each recommendation.
+
+#### Do we surface asset governance recommendations at the perspective level?
+
+No, we do not provide asset governance recommendations at the perspective level."
```
Comment (iamanmolm) [inline kb/cloud-cost-management/cloud-cost-management-faqs.md:308]: Remove quotes from the end.

**Review by iamanmolm — CHANGES_REQUESTED** [review]

File: kb/cloud-cost-management/cloud-cost-management-faqs.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (shashwat-harness) [reply kb/cloud-cost-management/cloud-cost-management-faqs.md:282]: I think the customer was asking about rightsizing the CPU and Memory limits based on expected demands. For instance, if we have Black Friday coming up next week, the customer is now interested in determining the ideal CPU and memory limits to configure, taking into account the expected surge in demand during that event.

[the remark this answers is not in the document]
Comment (iamanmolm) [reply kb/cloud-cost-management/cloud-cost-management-faqs.md:282]: We can also add, "In all supported recommendation we look at past data to generate recommendation."

[the remark this answers is not in the document]
Comment (shashwat-harness) [reply kb/cloud-cost-management/cloud-cost-management-faqs.md:282]: Done

**Review by iamanmolm — DISMISSED** [review]

File: kb/cloud-cost-management/cloud-cost-management-faqs.md (hunk 3 above)
Comment (shashwat-harness) [reply kb/cloud-cost-management/cloud-cost-management-faqs.md]: Done

**Review by iamanmolm — APPROVED** [review]

**Pull request merged**